src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -283,7 +285,8 @@ class LedgerMaster
     bool shouldAcquire(
```

Comment:
`shouldAcquire()` is only called from one place, and that is in LedgerMaster.cpp.  I think you should remove this declaration and have the implementation be a file scoped function in ledgerMaster.cpp.  More details in the LedgerMaster.cpp comments.

Reply:
fixed